documentation/src/main/asciidoc/topics/con_xsite_state_transfer.adoc

```diff
@@ -46,6 +46,12 @@ However, when using the asynchronous backup strategy, {brandname} can automatica
 When it detects that a backup location has come back online, and the network connection is stable, {brandname} initiates bi-directional state transfer between backup locations.
 For example, {brandname} simultaneously transfers state from **LON** to **NYC** and **NYC** to **LON**.
 
+[NOTE]
+====
+To avoid temporary network disconnects triggering state transfer operations, there are two conditions that backup locations must meet to go offline.
+The status of a backup location must be offline and it must not be included in the cross-site view with JGroups RELAY2.
+====
```

Comment:
@pruivo Hey I hope it's ok to be a little bit bold and sneak this in here. We discussed the two conditions (jgroups + infinispan) that need to be met before state transfer is triggered when sites come back online. I changed the wording a little. Hopefully didn't alter meaning. :grin: